storage/pom.xml

```diff
@@ -47,6 +46,21 @@
             <artifactId>zipkin-sender-kafka</artifactId>
             <version>${zipkin-sender-kafka.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.linecorp.armeria</groupId>
+            <artifactId>armeria</artifactId>
+            <version>0.90.3</version>
+        </dependency>
+        <dependency>
```

Comment:
consider jackson as that's less types of json libraries. not terribly important though